src/modules/70-pipeline/components/ExecutionActions/__tests__/ExecutionActions.test.tsx

```diff
@@ -28,6 +28,7 @@ jest.mock('services/pipeline-ng', () => ({
   useHandleStageInterrupt: jest.fn(() => ({
     mutate: jest.fn()
   })),
+  useGetExecutionData: jest.fn(() => ({})),
```

Comment:
For things like this, I like to use `mockReturnValue`
```suggestion
  useGetExecutionData: jest.fn().mockReturnValue({}),
```

Just an opinion, fine as-is